junit-vintage-engine/src/main/java/org/junit/vintage/engine/discovery/TestClassCollector.java

```diff
@@ -48,7 +48,11 @@ void addFiltered(Class<?> testClass, RunnerTestDescriptorAwareFilter filter) {
 	}
 
 	private Stream<TestClassRequest> completeRequests(Predicate<? super Class<?>> predicate) {
-		return completeTestClasses.stream().filter(predicate).map(TestClassRequest::new);
+		// @formatter:off
```

Comment:
Ah yes my bad, I added an extra filter here in an initial try, but forgot to remove the formatting